### DISCUSSION TITLE
Add ARM32 support

### DISCUSSION THREAD

**Discussion by EchoPouet** [prefix-dev/pixi#789] [Ideas]
Hello,

Can you support ARM32 processor soon ? I'm working on RasPi 2 with armv7 processor.

Thanks.

**Comment by ruben-arts** [comment]
That might be a hard target for us to support. We have no good way of testing this and there are almost no packages in `conda-forge` that support that platform. E.g. not even python: https://prefix.dev/channels/conda-forge/packages/python

What is your specific use-case? 

You're of-course free to try to compile `pixi` yourself, and let us know of your progress.

**Comment by EchoPouet** [comment]
I understand, if Python 3.XX in conda doesn't support arm32; it's hard to you to support this processor.

I'll use `pip` but it's sad :(

**Reply by ruben-arts** [reply]
Yeah exactly, would have loved to tell you a different story but its not available in the near future.

**Comment by EchoPouet** [comment]
I should ask CondaForge why it doesn't support this architecture, which is still supported by Linux distributions.

**Comment by lucascolley** [comment]
I don't think this is actionable without wider conda-forge support
